Cue/app/tabs/library/CardListView.js

```diff
@@ -13,6 +13,7 @@ import ListViewHairlineSeparator from '../../common/ListViewHairlineSeparator'
 type Props = {
   cards: Array<Card>,
   isFiltering: boolean,
+  flagCard?: (cardUuid: string, flag: boolean) => any,
```

Comment:
Yeah, if I annotate the type with `?` instead of the prop name and then don't specify the prop when creating the component, Flow will complain.